charts/spire/charts/spiffe-csi-driver/values.yaml

```diff
@@ -1,3 +1,6 @@
+# -- Set the csi driver name deployed to Kubernetes.
+pluginName: csi.spiffe.io
```

Comment:
Should this be `{{ .Values.pluginName }}`?

Reply:
No in values file this should be the real value. Templates only go inside the templates folder.